src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -646,14 +646,17 @@ public final Response getGameboardsByCurrentUser(@Context final HttpServletReque
      *            - so that we can find out the currently logged in user
      * @param gameboardId
      *            - So that we can look up an existing gameboard to modify.
+     * @param gameboardTitle
+     *            - So that the gameboard can ebe persisted to the user account with a custom
+     *              title (used by the CS question finder to name temporary gameboards)
      * @return a Response containing a list of gameboard objects or containing a SegueErrorResponse.
      */
     @POST
-    @Path("gameboards/user_gameboards/{gameboard_id}")
+    @Path("gameboards/user_gameboards/{gameboard_id}/{gameboard_title}")
```

Comment:
It's not really, since the gameboard is temporary and not yet persisted to the account - if I make a call to the updateGameboard endpoint, the board gets saved to the account by default, which isn't what we want. This change allows the temporary gameboard to be assigned a new title on link - perhaps a new endpoint `saveTemporayGameboard` is needed so that the `linkUserToGameboard` just does what would be expected.